app/src/store/getters.ts

```diff
@@ -543,6 +543,7 @@ export const getEntityTypesBC = (state: StateIF): EntityI[] => {
 export const getEntityTypesXPRO = (state: StateIF): EntityI[] => {
   let _entityTypesXproData = EntityTypesXproData
   if (isLocationCA(state)) {
+    _entityTypesXproData = _entityTypesXproData.filter(ent => ent.value !== EntityTypes.XUL)
     _entityTypesXproData = _entityTypesXproData.filter(ent => ent.value !== EntityTypes.RLC)
```

Comment:
Won't like 547 overwrite the value set in line 546?

Reply:
No, it won't overwrite the value set in line 546